docs/guides/how_to_guides/validation/how_to_update_data_docs_as_a_validation_action.rst

```diff
@@ -65,6 +65,12 @@ Steps
     2. Run the Checkpoint and verify that no errors are thrown. You can run the Checkpoint from the CLI as explained :ref:`here<how_to_guides__validation__how_to_run_a_checkpoint_in_terminal>` or from Python, as explained :ref:`here<how_to_guides__validation__how_to_run_a_checkpoint_in_python>`.
     3. Check your configured Data Docs sites to confirm that a new Validation Result has been added.
 
+Additional notes
+----------------
+
+The ``UpdateDataDocsAction`` generates an HTML file for the latest validation result and updates the index page to link to the new file. It does not perform a full rebuild of Data Docs sites. This means that every time an existing Expectation Suite is edited (or a new Expectation Suite is created), you should run full Data Docs rebuild (via CLI's ``great_expectations docs build`` command or by calling ``context.build_data_docs()``) for the current state of that Expectation Suite to be reflected in Data Docs.
```

Comment:
We currently do now re-render the expectation pages in the `UpdateDataDocsAction` - see PR #1990. So maybe change this to 
```suggestion
The ``UpdateDataDocsAction`` generates an HTML file for the latest validation result and updates the index page to link to the new file, and re-renders expectation suite pages. It does not perform a full rebuild of Data Docs sites. This means that if you wish to render older Validation Results, you should run full Data Docs rebuild (via CLI's ``great_expectations docs build`` command or by calling ``context.build_data_docs()``).
```